### DISCUSSION TITLE
可以加个通个cloudfare的api把优选最快的ip换到节点域名里面吗？

### DISCUSSION THREAD

**Discussion by iamsammajor** [XIU2/CloudflareSpeedTest#479] [讨论]
可以加个功能通个cloudfare的api把优选最快的ip换到节点域名里面吗？这样子就可以在我的htpc里面定时运行，然后得到ip，然后自动更新到CF上，达到某程度上的一劳永逸。。。。。。。

现在都是wrt设置好节点，然后测速然后手动到CF更新域名的DNS

还是有什么更快的方法？我是用AWS的CF

**Comment by XIU2** [comment]
参考 #481
